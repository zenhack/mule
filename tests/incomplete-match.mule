#XFAIL: match not implemented
match Ok {} with
  | Err x -> x
end
