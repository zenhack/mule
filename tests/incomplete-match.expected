
Desugared:

	((match ((Err ($6 ((fn x x) $6))))) ((fn x (Ok x)) {}))
Type error: could not instatiate rigid type variable
